pkg/apis/operator/v1alpha1/apiserver_types.go

```diff
@@ -22,6 +22,8 @@ import (
 
 // APIServerSpec defines the desired state of APIServer
 type APIServerSpec struct {
+	// TLSSecretRef referencing the TLS serving credentials for the API server
```

Comment:
```suggestion
	// TLSSecretRef references the TLS certificate and private key for serving the KubeCarrier API.
```
I wanted to get rid of `credentials` here, as TLS credentials are usually things like client certs and I don't want any confusion here.